source/org.jetbrains.kotlinx/kotlinx-serialization-protobuf-jvm/Additions/KotlinX.Serialization.Protobuf.ProtoNumberImpl.cs

```diff
@@ -0,0 +1,16 @@
+namespace KotlinX.Serialization.Protobuf 
+{
+
+	public sealed partial class ProtoNumberImpl
+    {
```

Comment:
Tabs-vs-spaces here and elsewhere.  Tabs should be used consistently.

Additionally, please create a separate PR to add a `.editorconfig`, possibly copying [`dotnet/android/.editorconfig`](https://github.com/dotnet/android/blob/e40d395c76a27cf05d06168e192c7a05119ffea2/.editorconfig).